test/test_site.rb

```diff
@@ -14,14 +14,20 @@ class TestSite < JekyllUnitTest
 
     should "have an array for plugins if passed as a string" do
       site = Site.new(site_configuration({ "plugins_dir" => "/tmp/plugins" }))
-      assert_equal ["/tmp/plugins"], site.plugins
+      array = Utils::Platforms.really_windows? ? ["C:/tmp/plugins"] : ["/tmp/plugins"]
```

Comment:
the path is not important here actually. This step and the next is to simply test if **plugins** are correctly detected **as specified**.
e.g. if were to set `plugins_dir => "/apps/bazinga"`, the array returned would be:
```shell
["C:/apps/bazinga"] # on Windows and AppVeyor CI
["/apps/bazinga"] # on Unixoids and Travis CI
```